egs/sre16/v1/local/make_musan.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
```

Comment:
if this script could trivially be upgraded to python3, that would  be great;
I now prefer python3 for new scripts.